### PR TITLE
Bump to new java sdk for bug fixing

### PR DESCRIPTION
**Pull request by zhang-hua**
# Type of change? 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

...

**Checklist:**

- [x] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by zhang-hua** [comment]
@dluc , the commits include fixes for bug #63, #62,  #55(duplicate #38), #54 (duplicate #36) and #26.

File: app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/runtime/Uptime.java [hunk 1]
```diff
@@ -41,6 +41,6 @@ public static Duration getDuration() {
      * @return A randomly generated ID used to identify the process in the logs.
      */
     public static String getProcessId() {
-        return processId;
+        return "WebService." + processId;
```
Comment (timlaverty) [inline app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/runtime/Uptime.java]: should this be IoTHubManager instead of WebService?

Comment (zhang-hua) [reply app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/runtime/Uptime.java]: This DotNet version is using this value.

Comment (zhang-hua) [reply app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/runtime/Uptime.java]: DotNet code is here: https://github.com/Azure/iothub-manager-dotnet/blob/master/WebService/Runtime/Uptime.cs#L17

Comment (dluc) [reply app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/runtime/Uptime.java]: "WebService" 👍 

@timlaverty within a microservice we can have multiple processes, e.g. "WebService" and "xyzAgent", so we use this to distinguish which process

File: test/com/microsoft/azure/iotsolutions/iothubmanager/services/JobsTest.java [hunk 2]
```diff
@@ -53,31 +55,47 @@ public static void tearDownOnce() {
...
-    // Ignore this test temporarily due to bug of SDK can not accept twin without
-    // Device ID. Otherwise it will always complains "IllegalArgumentException:
-    // Device ID cannot be null or empty". Will enable it once the bug is fixed.
-    @Ignore
-    @Test(timeout = 100000)
+    @Test(timeout = 310000)
     @Category({IntegrationTest.class})
     public void scheduleTwinJobAsyncTest() throws Exception {
-        String jobId = "unitTestJob" + batchId;
+        String jobIdPrefix = "IntegrationTestTwinJob" + batchId;
```
Comment (dluc) [inline test/com/microsoft/azure/iotsolutions/iothubmanager/services/JobsTest.java]: nit: use the test name as a prefix, so it's easy to know where the devices come from

File: test/com/microsoft/azure/iotsolutions/iothubmanager/services/JobsTest.java [hunk 3]
```diff
@@ -53,31 +55,47 @@ public static void tearDownOnce() {
...
+            }
+        }
+        System.out.println(String.format("Warning: passed this test finally because of job throttled for %d time", MAX_RETRIES));
     }
 
     @Test(timeout = 310000)
     @Category({IntegrationTest.class})
     public void scheduleMethodJobAsyncTest() throws Exception {
-        String jobId = "unitTestJob" + batchId;
+        String jobIdPrefix = "IntegrationTestMethodJob" + batchId;
```
Comment (dluc) [inline test/com/microsoft/azure/iotsolutions/iothubmanager/services/JobsTest.java]: same here, the prefix should be "scheduleMethodJobAsyncTest"

**Pull request merged**